api/api/accounts/urls.py

```diff
@@ -5,4 +5,5 @@
 urlpatterns = [
   path('', views.CreateUserView.as_view()),
   path('<int:pk>/', views.RetrieveUpdateUserView.as_view()),
+  path('roles/', views.ListUserRoles.as_view()),
```

Comment:
url should be `<int:pk>/roles/`

And should return roles for the user with pk in the url path.

Reply:
Is this even needed ? Current implementation get user if from `request.user` which is more reliable. We can do this but it would cause more exceptions to handle.